### DISCUSSION TITLE
use as "alarms display"

### DISCUSSION THREAD

**Discussion by toog** [fbaligand/kibana-enhanced-table#193] [Ideas]
Have you an evolution planned or is it already possible to use enhanced_table to display documents tagged like alarm, and have possibility to validate them by writing back to elastic with a modification of the document tag ...

**Comment by fbaligand** [comment]
Well, it is not possible yet, nor planned to let the plugin “write” data to Elasticsearch.
The plugin aims to view data.
You can only add a computed column with a dynamic link that call some service api that do the job. But you have to write the service api.
